src/constants.ts

```diff
@@ -1,62 +1,102 @@
 // mainnet btc address validation regex
-export const BTC_MAINNET_REGEX = /\b([13][a-km-zA-HJ-NP-Z1-9]{25,34}|bc1[ac-hj-np-zAC-HJ-NP-Z02-9]{11,71})\b/;
+const BTC_MAINNET_REGEX = /\b([13][a-km-zA-HJ-NP-Z1-9]{25,34}|bc1[ac-hj-np-zAC-HJ-NP-Z02-9]{11,71})\b/;
 // btc transaction validation regex
-export const BTC_TRANSACTION_ID_REGEX = /[a-fA-F0-9]{64}/;
+const BTC_TRANSACTION_ID_REGEX = /[a-fA-F0-9]{64}/;
 
 // regex for validating input strings as numbers
-export const NUMERIC_STRING_REGEX = /^[0-9]+([.][0-9]+)?$/;
+const NUMERIC_STRING_REGEX = /^[0-9]+([.][0-9]+)?$/;
 
-export const BITCOIN_NETWORK = (process.env.REACT_APP_BITCOIN_NETWORK || 'testnet') as
+const BITCOIN_NETWORK = (process.env.REACT_APP_BITCOIN_NETWORK || 'testnet') as
   | 'mainnet'
   | 'testnet'
   | 'regtest';
-export const BITCOIN_REGTEST_URL = process.env.REACT_APP_BITCOIN_REGTEST_URL || 'http://localhost:3002';
+const BITCOIN_REGTEST_URL = process.env.REACT_APP_BITCOIN_REGTEST_URL || 'http://localhost:3002';
 
-export const STORE_NAME = 'pbtc-store-2';
+const STORE_NAME = 'pbtc-store-2';
 
-export const BTC_ADDRESS_REGEX =
+const BTC_ADDRESS_REGEX =
   BITCOIN_NETWORK === 'mainnet' ?
     BTC_MAINNET_REGEX :
     BITCOIN_NETWORK === 'testnet' ?
       BTC_TESTNET_REGEX :
       BTC_REGTEST_REGEX;
 
-export const PARACHAIN_URL = process.env.REACT_APP_PARACHAIN_URL || 'ws://127.0.0.1:9944';
-export const DEFAULT_ACCOUNT_SEED = process.env.REACT_APP_DEFAULT_ACCOUNT_SEED;
-export const FAUCET_URL = process.env.REACT_APP_FAUCET_URL || 'http://localhost:3035';
+const PARACHAIN_URL = process.env.REACT_APP_PARACHAIN_URL || 'ws://127.0.0.1:9944';
+const DEFAULT_ACCOUNT_SEED = process.env.REACT_APP_DEFAULT_ACCOUNT_SEED || '//Alice';
+const FAUCET_URL = process.env.REACT_APP_FAUCET_URL || 'http://localhost:3035';
 
-export const STATS_URL = process.env.REACT_APP_STATS_SERVER_URL || 'http://localhost:3007';
+const STATS_URL = process.env.REACT_APP_STATS_SERVER_URL || 'http://localhost:3007';
 
-export const FEEDBACK_URL = 'https://forms.gle/2eKFnq4j1fkBgejW7';
+const FEEDBACK_URL = 'https://forms.gle/2eKFnq4j1fkBgejW7';
 
-// ######################################
-// STAKED RELAYER
-// ######################################
-export const STAKED_RELAYER_OK = 'Ok';
-export const STAKED_RELAYER_OFFLINE = 'Offline';
-export const STAKED_RELAYER_SLASHED = 'Slashed';
+// FIXME: hacky workaround to get the right ss58 prefix. Should be fetched at runtime instead
+// Possible example below:
+//       // Load the basic bridge data without depending on interbtc-api
+//      const api = await ApiPromise.create({ provider: new WsProvider(constants.PARACHAIN_URL) });
+//      // Default ss58 prefix is 42 for generic substrate chains
+//      const rawSs58Format = await (await api.rpc.system.properties()).ss58Format;
+//      const ss58Format = parseInt(rawSs58Format.unwrapOr('42').toString());
+let ss58Format;
```

Comment:
minor: this is why the exports suddenly stopped working - as long as the file is all exports it will work, but adding the function here triggered the compiler error. I think this is fine though - declaring and exporting en masse is consistent with much of the rest of the project, and when we implement the fix we can move the function out to a utility file and leave this file as exports only.

Reply:
@nud3l 
Yes, it's due to ESLinting set-up. I configured like so consistency.

Reply:
Agree! Just added here for simplicity now.